Source/WebInspectorUI/UserInterface/Controllers/JavaScriptRuntimeCompletionProvider.js

```diff
@@ -91,6 +91,20 @@ WI.JavaScriptRuntimeCompletionProvider = class JavaScriptRuntimeCompletionProvid
         return JavaScriptRuntimeCompletionProvider.__cachedCommandLineAPIKeys;
     }
 
+    // Public
+
+    clearCache()
```

Comment:
i think this needs a more specific name like `clearCachedPropertyNames` or something so that it's more clear what it's doing